packages/react-charting/src/components/AreaChart/AreaChartRTL.test.tsx

```diff
@@ -272,9 +281,48 @@ describe('Area chart rendering', () => {
       },
       undefined,
       undefined,
-      !isTimezoneSet(tzIdentifier),
+      !(isTimezoneSet(tzIdentifier) && isTestEnv()),
     );
   });
+
+  testWithWait(
```

Comment:
This entire code block containing repetitive code can be refactored as follows:
const testCases = [
  ['when tick Values is given', { data: chartDataWithDates, tickValues, tickFormat: '%m/%d' }],
  ['when tick Values not given and tick format is given', { data: chartDataWithDates, tickFormat: '%m/%d' }],
  ['when tick Values is given and tick format not given', { data: chartDataWithDates, tickValues }],
];

describe.each(testCases)('Should render the Area chart with date x-axis data %s', (testCase, props) => {
  testWithWait(
    `Should render the Area chart with date x-axis data ${testCase}`,
    AreaChart,
    props,
    container => {
      // Assert
      expect(container).toMatchSnapshot();
    },
    undefined,
    undefined,
    !(isTimezoneSet(Timezone.UTC) && isTestEnv()),
  );
});

Reply:
To be fixed in next iteration